content/docs/0.8.x/monitoring/prometheus/install/index.md

```diff
@@ -13,33 +13,72 @@ In order to evaluate the quality gates and allow self-healing in production, we
 
 ## Setup Prometheus
 
-After creating a project and service, you can setup Prometheus monitoring and configure scrape jobs using the Keptn CLI. Therefore you need to deploy the *prometheus-service*. 
+After creating a project and service, you can set up Prometheus monitoring and configure scrape jobs using the Keptn CLI.
 
-* Specify the version of the prometheus-service you want to deploy. Please see the [compatibility matrix](https://github.com/keptn-contrib/prometheus-service#compatibility-matrix) of the prometheus-service to pick the version that works with your Keptn.  
+Keptn doesn't install or manage Prometheus and its components. Users need to install Prometheus and Prometheus Alert manager as a prerequisite.
 
-    ```console
-    VERSION=<VERSION>   # e.g.: VERSION=0.3.5
-    ```
+Some environment variables have to set up in the prometheus-service deployment
+```yaml
+    # Prometheus installed namespace
+    - name: PROMETHEUS_NS
+    value: 'default'
+    # Prometheus server configmap name
+    - name: PROMETHEUS_CM
+    value: 'prometheus-server'
+    # Prometheus server app labels
+    - name: PROMETHEUS_LABELS
+    value: 'component=server'
+    # Prometheus configmap data's config filename
+    - name: PROMETHEUS_CONFIG_FILENAME
+    value: 'prometheus.yml'
+    # AlertManager configmap data's config filename
+    - name: ALERT_MANAGER_CONFIG_FILENAME
+    value: 'alertmanager.yml'
+    # Alert Manager config map name
+    - name: ALERT_MANAGER_CM
+    value: 'prometheus-alertmanager'
+    # Alert Manager app labels
+    - name: ALERT_MANAGER_LABELS
+    value: 'component=alertmanager'
+    # Alert Manager installed namespace
+    - name: ALERT_MANAGER_NS
+    value: 'default'
+    # Alert Manager template configmap name
+    - name: ALERT_MANAGER_TEMPLATE_CM
+    value: 'alertmanager-templates'
+```
 
-* To install the *prometheus-service*, execute: 
+#### Execute the following steps to install prometheus-service
 
-    ```console
-    kubectl apply -f https://raw.githubusercontent.com/keptn-contrib/prometheus-service/$VERSION/deploy/service.yaml
-    ```
+* Download the Keptn's Prometheus service manifest
```

Comment:
Please change to: `Download the manifest of the prometheus-service:`

Reply:
Done